dissect/target/filesystem.py

```diff
@@ -1300,6 +1300,8 @@ def symlink(self, src: str, dst: str) -> None:
 class LayerFilesystem(Filesystem):
     __type__ = "layer"
 
+    mounts: dict = {}
```

Comment:
Is this addition only for the unit test? I'd prefer finding a more suitable way to mock in that case :smile:.

Reply:
My last suggestion seems to work?